### DISCUSSION TITLE
Student ID card with backside

### DISCUSSION THREAD

**Discussion by wolflu05** [community/community#56707] [GitHub Education] [answered]
### Hi everyone,

Hello I’m trying to sign up for GitHub education. I tried it yesterday using the iOS scan function. Today got rejected then tried taking pictures. Again 10s later got rejected. Everytime got the following response:

> Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on May 29, 2023 for [@wolflu05](https://github.com/wolflu05).
> - Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

The problem is with my ID card. My name is on one side and the school stamp and sign on the other side. So I‘m not able to provide it with one picture so I used an image editing program and put both photos/Scans side by side. Unfortunately everytime getting the same error. Any idea how I can get this solved?

EDIT: also tried bending the paper so that school name, my name and school stamp + valid date is on the same picture made directly with the photo function => got some different reply:

> Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on May 29, 2023 for [@wolflu05](https://github.com/wolflu05).
> - Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.

**Comment by jge162** [comment, marked as answer]
Couple questions, are you using your school email as your main email on your GitHub account. Second try using transcripts. That’s how I was approved and many others I’ve helped here. Report back after you resubmit. Good luck.

**Reply by wolflu05** [reply]
Unfortunately our school doesn't offer email addresses for their students, so no, I'm using my private email.

How does transcripts offer a date? Usually they contain grades for a finished period? And does GitHub also accept blurring the grades? I want to be a bit privat.

**Reply by jge162** [reply]
I don’t see the need to blur grades honestly. It’s a secure upload and used to approve you. Transcripts show you are currently attending the school and school name. It has worked for many people. 

And if you don’t have school email I’ve seen people approved without one. So that’s should be ok.

**Reply by wolflu05** [reply]
Why do GitHub need the grades for verification? Yeah it is a secure upload, but I'm not sure what GitHub does with the uploaded data? Get this saved to GitHub's servers? For how long?

**Reply by jge162** [reply]
They do not need the grades (I’m pretty sure). I don’t know if they will accept an altered document. I have no idea. Most organizations do not accept altered documents. But give it a try. What does it hurt to try.

**Reply by wolflu05** [reply]
And second question. My transcript is in German, does this work too? There is some more text on it than my two sided Id card.

**Reply by jge162** [reply]
I do not know if they will take German. But you can try and if not use translated version. Hoping it works for you. As it worked for me using transcript’s.

**Reply by wolflu05** [reply]
What is a translated version of a transcript? Our school only hands them out in German.

**Reply by jge162** [reply]
Use a free translator online.

**Reply by wolflu05** [reply]
Thanks for the tips, I'll try it.

**Reply by wolflu05** [reply]
Worked, very fast this time. (I cut some white pieces of paper to cover the grades and it went through.)

**Reply by jge162** [reply]
glad to hear.

**Comment by miditkl** [comment]
I have the same problem, but my ID is translated (german, english and french i guess). But I have no chance to upload it, since its having a backside. How can I upload it now?

**Reply by wolflu05** [reply]
Try using another document. School transcripts worked for me, see the other conversation. I got approved in less than a few minutes. Now waiting to receive the benefits.

**Reply by miditkl** [reply]
Nvm, I tried it with my phone, now everything is unlocked. They just made it more complicated for us, to just verify. Everything is automated and no more manually checked ig. But anyways thx.

**Reply by wolflu05** [reply]
Good that it worked for you. Have fun. Everything is unlocked already? Doesn't it say to you it can take a while between your approved and you receive the benefits?

**Reply by miditkl** [reply]
I got it immediatly.

**Reply by miditkl** [reply]
Not anymore, they took it 2 times from me...
